modernize-links.cmake

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2022 LAAS-CNRS
+#
+# Author: Guilhem Saurel
+#
+
+#.rst:
+# .. ifmode:: user
+#
+#   .. command:: MODERNIZE_LINK(target SCOPE <INTERFACE|PUBLIC|PRIVATE> TARGETS [targets...] LIBRARIES [libraries...]
```

Comment:
I would call it `MODERNIZE_TARGET_LINK_LIBRARIES`. It better shows the link with the CMake original signature.